pymc/distributions/continuous.py

```diff
@@ -336,6 +336,11 @@ def logcdf(value, lower, upper):
             ),
         )
 
+    def get_moment(value, size, lower, upper):
+        lower = at.full(size, lower, dtype=aesara.config.floatX)
```

Comment:
Shouldn’t the dtype be taken from the op instead of aesara’s config?

Reply:
```suggestion
        lower = at.full(size, lower, dtype=value.owner.op.inputs[2])
```